.github/workflows/CI.yml

```diff
@@ -0,0 +1,32 @@
+name: CI
+
+on: [push, pull_request]
+
+jobs:
+  test:
+    name: Julia ${{ matrix.version }} - ${{ matrix.os }} - ${{ matrix.arch }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        version:
+          - '1.3'
+          - '1'
```

Comment:
I don't think this actually runs the latest release version (i.e 1.5)  (unlike travis)

Reply:
I guess it now does. Maybe it was fixed, but I was sure it did not several months ago when transitioning HDF5.jl

Reply:
From the CI logs, it looks like it does — it reports downloading Julia v1.5.3: https://github.com/JuliaIO/MAT.jl/pull/145/checks?check_run_id=1449246460#step:3:6